assets/src/edit-story/components/fontPicker/pickerContainer.js

```diff
@@ -132,10 +147,54 @@ function FontPickerContainer({ value, onSelect, onClose }) {
   useFocusOut(ref, onClose, [onClose]);
 
   // Scroll to offset for current value
-  const currentOffset = fonts.findIndex(({ name }) => name === value);
+  const [currentOffset, setCurrentOffset] = useState(
+    fonts.findIndex(({ name }) => name === value)
+  );
 
-  // This is static for now, but with search and used fonts, this will change later
-  const matchingFonts = fonts;
+  const [matchingFonts, setMatchingFonts] = useState(fonts);
+
+  const [updateMatchingFonts] = useDebouncedCallback(
+    () => {
+      if (searchKeyword.trim() === '') {
+        setMatchingFonts(fonts);
+        return;
+      }
+      const _matchingFonts = fonts.filter(({ name }) =>
+        name.toLowerCase().includes(searchKeyword.toLowerCase())
+      );
+      setMatchingFonts(_matchingFonts);
+    },
+    250,
+    {},
+    [searchKeyword, fonts]
+  );
+
+  const handleSearchInputChanged = useCallback(
+    ({ target }) => {
+      setSearchKeyword(target.value);
+      updateMatchingFonts();
+    },
+    [updateMatchingFonts]
+  );
+
+  const handleInputFocus = useCallback(() => {
+    setCurrentOffset(-1);
+  }, []);
+
+  const handleKeyPress = useCallback(
```

Comment:
I feel that now we support arrow keys inside the component when opened, <kbd>arrow down</kbd> should also be an option to open the dropdown in the first place when focused in the design panel.

I know that this happens in `fontPicker.js` but that file hasn't been touched in this PR yet so the comment goes here instead. 🤷

Reply:
I think I get what this means now, when `FontPickerSelect` inside `fontPicker.js` has focus and the person hits <kbd>down</kbd> open the menu.